core/controllers/payload_validator.py

```diff
@@ -135,7 +135,6 @@ def convert_string_to_bool(param: str) -> Optional[Union[bool, str]]:
     'ExplorationEmbedPage',
     'ExplorationMaybeLeaveHandler',
     'ExplorationStartEventHandler',
```

Comment:
Hi @ruinan-liu as you noticed that Accountpage and PendingAccountDeletionpage handlers are no more in our codebase, can you please remove them from this list also (if they are present) ?
\https://github.com/oppia/oppia/issues/13162#issuecomment-922034630

Thanks